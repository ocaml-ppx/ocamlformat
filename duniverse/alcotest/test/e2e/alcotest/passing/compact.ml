open Alcotest_engine.Private.Utils

let () =
  let open Alcotest in
  let id () = () in
  run __FILE__
    [
      ( "test-a",
        List.init 200 (fun i ->
            test_case (Format.sprintf "Test case #%d" i) `Quick id) );
      ("test-b", [ test_case "Third test case" `Quick id ]);
      ("test-c", [ test_case "Fourth test case" `Slow id ]);
    ]
