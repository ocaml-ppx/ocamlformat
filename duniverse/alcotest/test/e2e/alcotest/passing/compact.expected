Testing `test/e2e/alcotest/passing/compact.ml'.
This run has ID `<uuid>'.

................................................................................
................................................................................
..........................................

